snap/snapcraft.yaml

```diff
@@ -2,13 +2,13 @@ name: fractal
 adopt-info: fractal
 grade: stable # must be 'stable' to release into candidate/stable channels
 confinement: strict
-base: core20
+base: core22
+version: 4.4.2
+icon: org.gnome.Fractal.svg
 architectures:
   - build-on: amd64
-
-layout:
-  /usr/share/fractal/resources.gresource:
-    symlink: $SNAP/usr/share/fractal/resources.gresource
+  - build-on: arm64
+  - build-on: armhf
```

Comment:
Please drop armhf, I'd rather not support that arch at this time

Reply:
Okay, no problem. Removing them